lib/gpu/lal_neighbor_gpu.cu

```diff
@@ -115,7 +115,7 @@ __kernel void kernel_calc_cell_counts(const unsigned *restrict cell_id,
 #define tagint int
 #endif
 #ifdef LAMMPS_BIGBIG
-#define tagint long long int
+#define tagint long
```

Comment:
Does CUDA/OpenCL/HIP support `int64_t`?

That may be a safer bet in the long run (pun intended). We are currently not building on Windows with BIGBIG and I am not aware of any other platform that have 32-bit long ints, so this is likely ok (for now).

Reply:
CUDA and HIP use int64_t, this line is the OpenCL code.

Reply:
Unlike CUDA and HIP, the OpenCL code is compiled at runtime by the driver and should follow the spec. The spec says `long` it's 64bit.

Reply:
excellent. that resolves this then.

Reply:
@akohlmey just for future reference: if the kernels would be compiled to OpenCL C++ instead of OpenCL C, one could include `<opencl_def>`, which defines `int64_t`. See https://www.khronos.org/registry/OpenCL/specs/2.2/html/OpenCL_Cxx.html. Trying to use `int64_t` in OpenCL C will result in a compilation error.
```
<kernel>:784:34: error: unknown type name 'int64_t
```